merchants/models.py

```diff
@@ -95,6 +95,20 @@ def needs_update_statuses(cls) -> list[OrderStatus]:
         ]
 
 
+class OrderStatusWithFree(models.TextChoices):
+    """Same as OrderStatus, but includes the 'FREE' option used for filtering reservations without payments."""
+
+    # Note: Enums cannot be subclassed, so we have to redefine all "original" members.
```

Comment:
Good that this comment is included, as i was about to suggest subclassing the original :D